navbar-service.js

```diff
@@ -28,6 +32,23 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.site = {
+    title: config.data.siteTitle,
+    brand: config.data.style.brand,
+    url: '/'
+  };
```

Comment:
Is there a reason we don't want to do `site` and `tenant` via the route.vars instead of in this service?

```js
$routeProvider.when({
  vars: {
    navbar: {
      site: { ... },
      tenant: { ... }
    }
  });
```